src/pgrn-condition.c

```diff
@@ -99,6 +101,10 @@ PGrnConditionDeconstruct(PGrnCondition *condition, HeapTupleHeader header)
 			{
 				condition->columnName = NULL;
 			}
+			else if (i == fuzzyMaxDistanceRatioIndex)
+			{
+				condition->fuzzyMaxDistanceRatio = -1.0;
```

Comment:
`0.0` means "disabled" in `grn_table_selector_set_fuzzy_max_distance_ratio()`.

```suggestion
				condition->fuzzyMaxDistanceRatio = 0.0;
```